src/fdm_sdk_inject/_api/models/spaces.py

```diff
@@ -0,0 +1,211 @@
+from typing import Any, Iterator, Sequence, Union, cast, overload
+
+from cognite.client._api_client import APIClient
+from requests import Response
+from rich import print
+
+from ...data_classes.models.spaces import ModelsSpace, ModelsSpaceList
+
+#
+# HINTS: used cognite/client/_api/assets.py as template for methods
+#       cognite/client/_api/transformations/jobs.py for folder-structure
+#
+
+
+#
+# helper
+#
+def is_identifier_sequence(identifier):
+    """Checks if parameter is list of element or string
+
+    Args:
+        identifier (str|Sequence[str]): identifier to check
+
+    Returns:
+        bool: True if it is a sequence
+    """
+    return isinstance(identifier, Sequence) and not isinstance(identifier, str)
+
+
+class ModelsSpacesAPI(APIClient):
```

Comment:
Do you implement missing parts within SDK? Just to make sure I understood you correctly

Reply:
yes
as there is no `Models` support in Cognite Python SDK, and I need some parts to make the `space` create/delete working